test/listr.test.ts

```diff
@@ -1,18 +1,19 @@
+import { afterEach, beforeEach, expect, test, vi } from 'vitest'
 import { wrapTask } from '../lib'
 import * as logging from '../lib/logging'
 
-jest.mock('../lib/logging')
+vi.mock('../lib/logging')
```

Comment:
love the migration to vitest

Reply:
🙇 

We had to, jest is so annoying with ESM 🙈